deploy/csi-operator.yaml

```diff
@@ -920,6 +920,22 @@ roleRef:
   name: openebs-cstor-csi-provisioner-role
   apiGroup: rbac.authorization.k8s.io
 
+---
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: openebs-csi-controller-critical
+value: 900000000
+globalDefault: false
+description: "This priority class should be used for the CStor CSI driver controller deployment only."
```

Comment:
If it is only for cStor then should we need to add `cStor` as suffix in name? Why do we need two Priority classes? If other engines also use the same priorityclass name then there will be conflicts(via helm install/kubectl create)

Reply:
done